hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestSparkConsistentBucketClustering.java

```diff
@@ -110,7 +115,7 @@ public void setup(int maxFileSize, Map<String, String> options) throws IOExcepti
         .withStorageConfig(HoodieStorageConfig.newBuilder().parquetMaxFileSize(maxFileSize).build())
         .withClusteringConfig(HoodieClusteringConfig.newBuilder()
             .withClusteringPlanStrategyClass(SparkConsistentBucketClusteringPlanStrategy.class.getName())
-            .withClusteringExecutionStrategyClass(SparkConsistentBucketClusteringExecutionStrategy.class.getName())
+            .withClusteringExecutionStrategyClass(single ? SINGLE_SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY : SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY)
```

Comment:
We should just keep one strategy if `SINGLE_SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY` is always better.

Reply:
> We should just keep one strategy if `SINGLE_SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY` is always better.

Yes, I will deprecated previous hash resizing execution in next pr.